src/components/layout/RightSectionWallet.vue

```diff
@@ -23,7 +23,7 @@
       <div class="not-bootstrap-row">
         <AeAmount :amount="balance" />
         <Dropdown
-          v-if="currencyDropdownOptions.length"
+          v-if="currencyDropdownOptions.length && balance"
```

Comment:
Would be nice to hide the dropdown in this case, but it is not related to #660 bug because obviously on the screenshots from there the balance is not 0, but the dropdown contains 0 currency amounts.